### DISCUSSION TITLE
Simple Package to Create Clover Boot Drive

### DISCUSSION THREAD

**Discussion by krasnit** [CloverHackyColor/CloverBootloader#529] [Q&A]
A few years ago I found a simple Clover package on the Internet that would create a Clover boot drive on a USB stick. It was meant for a Hackintosh, but I wanted to use it for booting Windows from a NvMe drive with an older non-UEFI  motherboard. There are plenty of NvMe adapters that plug into a PCIe slot.  It worked fine with some motherboards but not with others.   I have not been able to find that site, although I still have the package somewhere which is pretty old at this point and Clover has continued to be developed.  
However, I was also not satisfied with the messy idea of using an external USB stick to boot, so after spending considerable time and effort I was able to create a bootable Clover partition on an old mechanical hard drive using that package.  I do not recall how I managed to do that, but it took quite a bit of time.  I, like perhaps many others, have quite a few of those mechanical hard drives from installing SATA SSDs on older computers.  The Clover boot partition is quite small and is only used for booting the boot manager/UEFI driver, and the rest of the drive can be a large partition used as a data drive for various storage.  
In short, is there a simple up to date package anywhere that will allow me to easily install Clover on a SATA hard drive so that Windows (or any other OS) can boot off a NvMe drive with an older motherboard?

**Comment by Morbius01** [comment]
I don’t know of a simple Clover package that would enable you to install Clover on a SATA HDD, but you could just create a USB Clover key in a first stage, and then clone it to your SATA HDD.

I just succeded to clone my Clover USB key (which I created using BootDisk Utility v.023) to a 32GB SATA SSD, using the **ddrescue** utility in Linux. It works great, and I can now boot all my OSs installed on my (non bootable) 1TB NVMe SSD after launching Clover from my SATA SSD.

I do not have any spare SATA HDD to test, but I see no reasons why, after it worked on a SATA SSD, it would not work on a SATA HDD.
